src/wasm/wasm-emscripten.cpp

```diff
@@ -774,7 +827,10 @@ void AsmConstWalker::queueImport(Name importName, std::string baseSig) {
   auto import = new Function;
   import->name = import->base = importName;
   import->module = ENV;
-  import->type = ensureFunctionType(baseSig, &wasm)->name;
+  auto type = ensureFunctionType(baseSig, &wasm);
+  import->type = type->name;
+  import->result = type->result;
+  import->params = type->params;
```

Comment:
We need to fill in `import->result` and `import->params` since we end up calling `getSig` on the function. Without it, the function is thought to return `void` and take no arguments.

Somehow, it wasn't needed before.